set.mm

```diff
@@ -437504,10 +437504,10 @@ orthogonal vectors (i.e. whose inner product is 0) is the sum of the
 /* Elementary  geometry (extension)  */
 htmldef "LineM" as 'LineM';
  althtmldef "LineM" as 'Line<sub>M</sub>';
- latexdef "LineM" as "{\rm Line}_M";
+ latexdef "LineM" as "{\mathrm{Line}_M}";
```

Comment:
```suggestion
 latexdef "LineM" as "\mathrm{Line}_M";
```

Reply:
I cannot apply the suggestion... "This diff has recently been updated. Refresh and try again." but refreshing has no effect...

Reply:
I can see the button "commit suggestion". I tried several times but each time I get "_This diff has recently been updated - Refresh and try again_"